dev/.vscode/settings.json

```diff
@@ -30,5 +30,13 @@
   "typescript.referencesCodeLens.enabled": true,
   "telemetry.enableTelemetry": false,
   "telemetry.enableCrashReporter": false,
-  "window.zoomLevel": 0
-}
\ No newline at end of file
+  "window.zoomLevel": 0,
+  "python.formatting.provider": "autopep8",
+  "python.linting.pep8Args": [
+    "--max-line-length=100"
```

Comment:
We use 80 for the eslint config -- think it's worth doing the same for python?

Reply:
yeah why not. I don't know why I decided to split it down the middle between 80 and 120, to an unhappy compromise